proposed/event-manager.md

```diff
@@ -147,19 +148,19 @@ interface EventManagerInterface
     /**
      * Attaches a listener to an event
      *
-     * @param string $event the event to attach too
+     * @param string $event the event to attach to
      * @param callable $callback a callable function
      * @param int $priority the priority at which the $callback executed
-     * @return bool true on success false on failure
+     * @return bool true on success; false on failure
      */
     public function attach($event, $callback, $priority = 0);
 
     /**
      * Detaches a listener from an event
      *
-     * @param string $event the event to attach too
+     * @param string $event the event to attach to
```

Comment:
Isn't this should be detach or remove? or is it overwriting the event itself. Slow connection unable to send a PR.